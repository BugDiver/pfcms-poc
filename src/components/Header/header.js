import React from 'react';
import { graphql, useStaticQuery, Link } from 'gatsby';
import headerStyle from './header.module.scss';


const Header = () => {
    const data = useStaticQuery(graphql`
        query {
            site {
                siteMetadata {
                    title
                    description
                }
            }
        }
    `);

    return (
        <header>
            <h1 className={'f1 ' + headerStyle.h1} >
                <Link className={headerStyle.link} to="/">
                    {data.site.siteMetadata.title}
                </Link>
            </h1>
            <p>{data.site.siteMetadata.description}</p>
        </header>
    )
}

export default Header