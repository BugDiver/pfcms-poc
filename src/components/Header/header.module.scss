/* #### Generated By: http://www.cufonfonts.com #### */

@font-face {
  font-family: 'SEGA LOGO FONT';
  font-style: normal;
  font-weight: normal;
  src: local('SEGA LOGO FONT'), url('./SEGA.woff') format('woff');
}

.header {
  margin-bottom: 7rem;
  margin-top: 3rem;

  .h1 {
    font-family: 'SEGA LOGO FONT', sans-serif;
    font-weight: 200;
    color: #0ccac4;
    margin: 0.5rem;
  }

}