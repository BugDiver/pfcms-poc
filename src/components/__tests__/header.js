import React from "react"
import renderer from "react-test-renderer"
import Header from "../Header/header"

describe("Header", function () {
  it("should render heading", function () {
    const tree = renderer
      .create(
        <Header
          data={{ site: { siteMetadata: { title: "a", description: "b" } } }}
        />
      )
      .toJSON()

    expect(tree).toMatchSnapshot()
  })
})
