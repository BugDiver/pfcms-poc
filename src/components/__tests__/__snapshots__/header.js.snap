// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render heading 1`] = `
<header
  className="header"
>
  <h1
    className="f1 h1"
  >
    <a
      className="link"
      href="/"
    >
      a
    </a>
  </h1>
  <p>
    b
  </p>
</header>
`;
